spec/scenarios/commitments/minimum/in_advance/anniversary/monthly_spec.rb

```diff
@@ -130,7 +130,7 @@
         }
       )
 
-      Subscriptions::BillingService.new.call
+      Subscriptions::OrganizationBillingService.call(organization:)
       perform_all_enqueued_jobs
```

Comment:
I was expecting a lot more jobs to use the `perform_billing` helper. No need to change this tho.

Reply:
Haa, good catch! I think I will replace the calls with this. It's a good opportunity to improve the specs :slightly_smiling_face: